node_modules/gh-users/lib/api.js

```diff
@@ -295,6 +296,64 @@ var updateUser = module.exports.updateUser = function(ctx, userId, update, callb
 };
 
 /**
+ * Change a user's local password
+ *
+ * @param  {Context}        ctx                     Standard context containing the current user and the current app
+ * @param  {string}         userId                  The id of user for which to change the local password
+ * @param  {string}         newPassword             The new password for the user
+ * @param  {string}         [oldPassword]           The previous password for the user. This is only required when the current user is not an administrator
```

Comment:
I believe these should be `String` instead of `string`. Though I have been leaning more lately towards using JSDoc (syntax is much better documented) which uses lower-case for native types.